keel-web/src/main/kotlin/com/netflix/spinnaker/keel/rest/InteractiveNotificationCallbackController.kt

```diff
@@ -59,4 +63,22 @@ class InteractiveNotificationCallbackController(
         )
       )
   }
+
+  /**
+   * New endpoint for the new slack integration. This will be called from gate directly instead of echo.
+   */
+  @PostMapping(
+    path = ["/v2"],
```

Comment:
Consider making this `/notifications/callbacks/slack` for readability.

Reply:
what do you mean? the path is `/notifications/callback/v2` (as I didn't want to override the existing endpoint).

Reply:
I meant that `v2` doesn't really mean anything, and so was suggesting to use `slack` for that last bit. My understanding was that gate calls keel on this endpoint, and so you could make this path whatever you want, right?

Reply:
done!